__tests__/app.test.js

```diff
@@ -40,6 +40,33 @@ describe("/api/topics",()=>{
     })
 })
 
+describe("/api/topics",()=>{
+    test("GET 200 and the requested article by id", () => {
+        return request(app)
+            .get("/api/articles/1")
+            .expect(200)
+                .then(({body})=>{
+                    const {article} = body
+                    expect(article.title).toBe("Living in the shadow of a great man")
+                    expect(article.topic).toBe("mitch")
+                    expect(article.author).toBe("butter_bridge")
+                    expect(article.body).toBe("I find this existence challenging")
+                    expect(article.created_at).toBe("2020-07-09T20:11:00.000Z")
+                    expect(article.votes).toBe(100)
+                    expect(article.article_img_url).toBe("https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700")
+                })
+    })
+    test("GET 404 when given an invalid article_id",()=>{
```

Comment:
This is a valid but non-existent article id. An invalid id would be something like /api/articles/bananas - what happens if the person tries to access an invalid id? How should the server handle this?

Reply:
Hint: think about psql error codes